packages/applicant/src/middleware.page.ts

```diff
@@ -163,6 +163,18 @@ async function getApplicationStatusBySubmissionId(
   return await data.text();
 }
 
+async function getIsSubmissionSubmittedBySubmissionId(id: string, jwt: string) {
```

Comment:
Can this function name be simplified? I would stick to the boolean naming convention down below e.g `isSubmissionSubmitted`